index.html

```diff
@@ -601,9 +601,32 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 or local resources.</li>
 <li>The time immediately after the user agent finish establishing
 the connection to the server to retrieve the resource, otherwise.
-<p>The returned time MUST include the time interval to establish
-the transport connection, as well as other time intervals such as
-SSL handshake and SOCKS authentication.</p>
+
+<p>The returned time MUST include the time interval to establish the transport
+connection, as well as other time intervals such as SOCKS authentication. It
+MUST include the time interval to complete enough of the TLS handshake to
+request the resource. That is:</p>
+
+<ul>
+  <li>If the user agent used TLS False Start [[RFC7918]] for this connection,
```

Comment:
I was expecting a list of things that must be including after "That is:"... perhaps we can change the wording here since this list is more than a simple clarification.

Reply:
Just deleted "That is:". Let me know if this is clearer

Reply:
Thanks!